### DISCUSSION TITLE
Where are settings located ?

### DISCUSSION THREAD

**Discussion by DarkXero-dev** [GradienceTeam/Gradience#654] [Q&A] [answered]
Hello

I maintain my own Distro, I was wondering where are Gradience settings located once I apply a color scheme like Layan ? Am asking coz I want to include it on my Gnome release...

Thanks

**Comment by daudix** [comment, marked as answer]
Hello

First you should look at [this](https://github.com/GradienceTeam/Gradience#%EF%B8%8F-gradience-stopthemingmyapp-and-adwaita-developers)

Settings are located in `~/.config/gtk-3.0/gtk.css` and `~/.config/gtk-4.0/gtk.css`
